src/metabase/sync/interface.clj

```diff
@@ -97,6 +97,40 @@
 (def Fingerprint
   "Schema for a Field 'fingerprint' generated as part of the analysis stage. Used to power the 'classification' sub-stage of
    analysis. Stored as the `fingerprint` column of Field."
-  {(s/optional-key :global)       GlobalFingerprint
+  {(s/optional-key :version)      su/IntGreaterThanZero ; Fingerprints with no version key are assumed to have version of 1
```

Comment:
Yeah, that's a good point.

It probably makes more sense just to make `fingerprint_version` its own column for that reason. At any rate I didn't end up including the logic for updating fingerprints for fields that need it in this PR, I'm planning on doing that as a follow-on. So that's something I can do there